templates/app/models/index.hbs

```diff
@@ -1,7 +1,18 @@
 {{#if isMongoDB}}
 const mongoose = require('mongoose');
+{{#each models as |model|}}
+const {{model.modelName}} = require('./{{model.modelFile}}');
+{{/each}}
```

Comment:
Is it really maintainable to have to require all model files here.
What if I want to add a new model definition? Will I have to go in this file to require it manually?

Reply:
We decided to do as such to fit with the way we import / exports models in sequelize projects. All models are accessible through the index file everywhere in the app. 

I didn't find any clear way of exporting / importing every models with mongoose. So if you have any other idea, I take it.